DSE_5.1_Release_Notes.md

```diff
@@ -2,6 +2,36 @@
 DSE 5.1.x is compatible with Apache Cassandra&trade; 3.11 and adds additional production-certified changes, if any. Components that are indicated with an asterisk (&ast;) (if any) are known to be updated since the prior patch version.
 
 
+# Release notes for 5.1.31
+11 May 2022
+
+## Components versions for DSE 5.1.31
+ * Apache Solr™ 6.0.1.0.2882
+ * Apache Spark™ 2.0.2.42
+ * Apache TinkerPop™ 3.2.11-20210716-faea8d16
+ * Apache Tomcat® 8.5.75&ast;
+ * DSE Java Driver 1.8.3-dse+20201217
+ * Netty 4.0.54.1.dse
+ * Spark JobServer 0.6.2.240
+
+## 5.1.31 DSE Core
+* Ensure that sstables are not unnecessarily read for columns that are not selected. See CASSANDRA-16737. (Previously DB-4974). (DSP-22478)
```

Comment:
If we want to stick to the guideline (https://datastax.jira.com/wiki/spaces/EN/pages/3374972929/Release+notes+guideline), the sentence should start with the verb in the past tense.
How about sth like: `Improved reading logic to ensure sstables are not unnecessarily read for columns that are not selected`?